src/Helpers/miscHelper.js

```diff
@@ -90,3 +90,8 @@ export const setupFilters = (filters, meta, defaultFilters, apply) => {
 
 export const isTimestampValid = (stateTimestamp, actionTimestamp) =>
   actionTimestamp >= stateTimestamp;
+
+export const urlChangeTab = (urlPath, newTab) => {
+  const lastSlashIndex = urlPath.lastIndexOf('/');
+  return urlPath.substring(0, lastSlashIndex + 1) + newTab;
```

Comment:
should we use the react-router navigate function instead of mutating the URL pathname?

Reply:
We are not directly using this function to perform navigate, we are just calculating the URL path for the `<Link> `component. I believe using `<Link>` is even clearer than using `navigate()`.

But I agree, if this function was to edit the URL it would be worse than `navigate`.

Reply:
I have switched to `useNavigate` after all to save a few lines of code used to style the `Link`.